### PR TITLE
Add more instruction when install GnuPG

### PR DESCRIPTION
**Pull request by tuanphpvn**
Add more instructions fix this error:  version GPG_ERROR_1.0 not defined in file libgpg-error.so.0
References: https://bugs.archlinux.org/task/48994

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -39,7 +39,10 @@
     libgcrypt    (ftp://ftp.gnupg.org/gcrypt/libgcrypt/)
     libksba      (ftp://ftp.gnupg.org/gcrypt/libksba/)
     libassuan    (ftp://ftp.gnupg.org/gcrypt/libassuan/)
-
+  
+  Downgrade maybe required. For example, on Ubuntu 16.04 latest version of libgcrypt will not work. 
+  In that case we need downgrade the library.
```
Comment (JohnVonNeumann) [inline README]: Grammatical suggestion:

Downgrade maybe required. For example, on Ubuntu 16.04, the latest version of libgcrypt will not work. In this case you will need to downgrade the library.

I also have no other context but I'd recommend providing more context as to why it doesn't work, like a blog post or something like that.

Comment (tuanphpvn) [reply README]: Thank you so much.

**Comment by jeroen** [comment]
This is a read-only mirror. Please post your [patch](https://github.com/gpg/gnupg/pull/5.patch) to the [gnupg-devel](https://lists.gnupg.org/mailman/listinfo/gnupg-devel) mailing list.

**Comment by 0xF135DD133B2E1111** [comment]
fgnfdnd

**Comment by 0xF135DD133B2E1111** [comment]
bffxnfdn

**Pull request closed without merging**